cypress/integration/at_mentions/at_mentions_spec.js

```diff
@@ -230,6 +230,8 @@ describe('at-mention', () => {
             cy.get('@postMessageText').
                 find(`[data-mention=${receiver.username}]`).
                 should('not.exist');
+
+            cy.get('#sidebarItem_saepe-5').scrollIntoView().click({force: true});
```

Comment:
nit: Added missing step

Reply:
Another nit, no need to scroll into view if we are forcing the click.